### ISSUE TITLE
Add export of succinate from cells and mitochondria

### ISSUE THREAD

**Issue by johan-gson** [SysBioChalmers/Human-GEM#424]
### Description of the issue:
We should add export of succinate from cells and mitochondria.
There is some evidence for that, for example:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273505/
Also, that complex II can be run in reverse, which I assume implicitly says that succinate is exported:
https://www.science.org/doi/10.1126/science.abi7495
This is important in hypoxia, since fumarate turned into succinate will accept electrons as an alternative to oxygen.

#### Expected feature/value/output:
Succinate can be exported out of the cell, I guess first from mitochondria to cytosol, and then out of the cell, and then an exchange rxn.

**I hereby confirm that I have:**
- [X] Tested my code on my own computer for running the model
- [X] Done this analysis in the `main` branch of the repository
- [X] Checked that a similar issue does not exist already

**Comment by haowang-bioinfo** [comment]
@johan-gson sounds good

what specific changes do you suggest to start?

**Comment by feiranl** [comment]
There is a reaction for Succinate import  which is co-transport with Sodium in Recon with gene annotation  of "SLC13A2" or "SLC13A3", but not sure whether we should add sodium too.
http://bigg.ucsd.edu/models/Recon3D/reactions/HMR_9610
http://bigg.ucsd.edu/models/Recon3D/reactions/SUCCt4_2
http://bigg.ucsd.edu/models/Recon3D/reactions/SUCCt4_2

**Comment by mihai-sysbio** [comment]
@feiranl this is still available in the latest version of the model, see https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR09610

**Comment by feiranl** [comment]
Yes. It is just import. I think Johan wants an export rxn. Recon has three rxns with both import and export.

**Comment by johan-gson** [comment]
Hi, yeah sorry, I didn't focus so much on import, that seems to already exist. Good sign that the genes are the same. The export is the important thing for cases like hypoxia, although import of course also needs to be there as well since other cells need to take this up. I looked a bit at transport in/out of mitochondria, there are a lot of antiporters in the model for this. I did however have to explicitly add an export reaction to make my simulations work, and I think the sodium-dependent reactions should be added. An alternative is to make this (http://bigg.ucsd.edu/universal/reactions/SUCCtm) reversible, it is not now (see below).

I think the sodium should be added to the reactions, the recon3D reactions are probably right, so 2 Na ions in the reaction as well. 

Transport reactions today:

	MAR04854		fumarate[c] + succinate[m] <=> fumarate[m] + succinate[c]		ENSG00000183048
	MAR04862		Pi[m] + succinate[c] => Pi[c] + succinate[m]		ENSG00000183048
	MAR04863		succinate[c] + sulfate[m] <=> succinate[m] + sulfate[c]
	MAR04864		succinate[c] + sulfite[m] <=> succinate[m] + sulfite[c]
	MAR06287		citrate[c] + H+[c] + succinate[m] => citrate[m] + H+[m] + succinate[c]		ENSG00000100075
	MAR06290		H+[c] + isocitrate[c] + succinate[m] => H+[m] + isocitrate[m] + succinate[c]		ENSG00000100075
	MAR06294		cis-aconitate[c] + H+[c] + succinate[m] => cis-aconitate[m] + H+[m] + succinate[c]		ENSG00000100075
	MAR02220		citrate[c] + succinate[m] <=> citrate[m] + succinate[c]		ENSG00000100075
	MAR02413		isocitrate[m] + succinate[c] <=> isocitrate[c] + succinate[m]		ENSG00000100075
	MAR02418		cis-aconitate[m] + succinate[c] <=> cis-aconitate[c] + succinate[m]		ENSG00000100075

**Comment by johan-gson** [comment]
I think then that this will be solved automatically when Human2 imports changes from Recon3D, correct? Let's check if the GPRs match when that is done, and after that we can close this issue.

**Comment by edkerk** [comment]
The mitochondrial side is already covered: succinate can leave the mitochondria through the reversible SLC25A10 and SLC25A1 antiporters (exchange with malate, fumarate, sulfate, sulfite, or citrate). But you were right that cellular export was missing: the three plasma-membrane succinate transporters (MAR09610, MAR04939, MAR04938) are all Na-coupled and import-only (irreversible), so cytosolic succinate had no way out, and the Recon3D import did not add an export.

Added a cellular export reaction, `MAR20192`: `succinate[c] + H+[c] <=> succinate[e] + H+[e]`, GPR `SLC16A1 or SLC16A3`, reversible, modelled on the existing lactate MCT reaction and based on the MCT-mediated succinate export you cited ([PMID:35363247](https://pubmed.ncbi.nlm.nih.gov/35363247/)). Import via SLC13A2/SLC13A3 already existed, so the pair is now complete and succinate can be secreted through the existing exchange reaction MAR09415.
